### ISSUE TITLE
Pulmonary capillary angioectasia

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6523]
**Preferred term label:** Pulmonary capillary angioectasia

**Synonyms**
Pulmonary capillary dilatation

**Definition (free text, please give PubMed ID)**
Focal accumulation of dilated pulmonary capillaries

**Parent term (use hpo.jax.org/app)**
Pulmonary capillary hemangiomatosis (HP:0005954) or Abnormality of the pulmonary vasculature (HP:0004930)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Pulmonary Veno-occlusive Disease 1 OMIM:265450

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Should this be related to Telangioectasia?
https://hpo.jax.org/app/browse/term/HP:0001009

**Comment by eswietlik** [comment]
I think rather not, telangiectasia refers to skin and mucose membranes.

**Comment by pnrobinson** [comment]
added, thanks!
